public/app/features/scenes/scenes/variablesDemo.tsx

```diff
@@ -45,37 +45,14 @@ export function getVariablesDemo(): Scene {
           text: '',
           options: [],
         }),
-        new ConstantVariable({
```

Comment:
We should remove the import for ConstantVariable since we don't use it anymore. It is breaking the build linting actually :) 
```js
import { ConstantVariable } from '../variables/variants/ConstantVariable';
```